pkg/kn/commands/service/list_mock_test.go

```diff
@@ -82,6 +82,146 @@ func TestServiceListAllNamespaceMock(t *testing.T) {
 	r.Validate()
 }
 
+func TestListEmptyMock(t *testing.T) {
+	// New mock client
+	client := knclient.NewMockKnClient(t)
+
+	// Recording:
+	r := client.Recorder()
+
+	r.ListServices(knclient.Any(), &v1alpha1.ServiceList{}, nil)
+
+	output, err := executeServiceCommand(client, "list")
+	assert.NilError(t, err)
+	assert.Assert(t, util.ContainsAll(output, "No", "services", "found"))
+
+	r.Validate()
+}
+
+func TestGetEmptyMock(t *testing.T) {
```

Comment:
should we rename this to `TestListEmptyWithArgMock` ?

Reply:
yes. makes sense. renamed it